src/components/TextInput/TextInput.vue

```diff
@@ -256,7 +257,7 @@ export default {
     selectedOptions () {
       return this.$slots.selectedOptions;
     },
-    showClearButton () {
+    showwithClearButton () {
```

Comment:
nit: Can the 'w' in 'with' be capitalized: `showWithClearButton`?

Reply:
totally, missed it, thanks!